Main.py

```diff
@@ -0,0 +1,48 @@
+# FIRST-NAME_LAST-NAME.py
+# This file is a template and only an example. Please customize as needed.
+
+
+# Step 1: Initialize Book Inventory
```

Comment:
These comments are meant to be placed before your changes. Move these comments to the expected code block.

Reply:
I add it right now

Reply:
@Omarmanassrah 
Please move the right comments before each step, example 
Step 1: Initialize Book Inventory 
Here write your code 
`books= [...]`

Step 2: ....

Reply:
I resolve it @Mohammed-samaheen are you approved to it